challenge.rb

```diff
@@ -22,7 +23,7 @@ def is_of_age? age, minimum_age
 # Remember: `p`, `print`, and `puts` don't return their arguments.
 
 def okay_to_drink? age, drinking_age
-  # Replace this comment with your code.
+  check >=0 ? "Come on in" : "I'm sorry " + age + " ins't old enough to drink. The minimun age is " + drinking_age +"."
```

Comment:
The ternary is appropriate, but the string interpolation is off. Unlike JavaScript, Ruby does not implicitly convert a number to a string when the two are added together. We need to either explicitly convert the numbers to strings using `.to_s` or use the built-in string interpolation syntax `"blah blah #{interpolated_value}"`